workflow/Tiltfile

```diff
@@ -0,0 +1,79 @@
+load('ext://uibutton', 'cmd_button')
+
+def set_work_cmds(flow_name, res_name, work_cmds):
+    cmds = ['./write_workflow.sh', flow_name, res_name]
+    for i in work_cmds:
+        cmds.append(" ".join(i))
```

Comment:
I think you might need to do something like `cmds.append(" ".join(shlex.quote(arg) for arg in i))` to prevent escaping issues.

(Try having a work_cmd that's something like `['/bin/sh', '-c', 'echo "Hello World"']` for example - I _think_ that might be sufficient to break it)